src/WebJobs.Extensions.ApiHub/Table/TableClientValueProvider.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Reflection;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+
+namespace Microsoft.Azure.WebJobs.Extensions.ApiHub.Table
+{
+    internal class TableClientValueProvider : IValueProvider
```

Comment:
Yeah, for bindings people usually hide the IValueProvider and other simple classes as nested classes to avoid having too many files. If you *really* want the separate classes, so be it, but all the other bindings are using nested classes (hint hint :))